packages/components/src/Truncate/Truncate.tsx

```diff
@@ -58,10 +65,17 @@ export const Truncate: FC<TruncateProps> = ({ children, width = '100%' }) => {
     width: 'auto',
   })
 
+  const className = join([domProps.className, propsClassName], ' ')
```

Comment:
I'd just use built-in join here. 

```suggestion
  const { className, otherDomProps } = domProps
  const className = [className, propsClassName].join(' ')
```